### DISCUSSION TITLE
Collection Item Dropdown support choose multiple items

### DISCUSSION THREAD

**Discussion by loongmxbt** [directus/directus#23165] [Feature Requests]
### Describe the Improvement

Now Collection Item Dropdown only support choosing one item. I think it's useful to enable multiple selection.

**Comment by danmaze** [comment]
You can create a one-to-many relational field to support this - will need a corresponding separate collection to hold this relation which some might not like. However, I think this approach keeps the data model more normalized than if an array of item IDs were used.

**Comment by loongmxbt** [comment]
Hi @danmaze, I think this feature is an alternative to "many-to-many" scenarios. One-to-Many cannot be used alone and need another M2O to correspond with. For example, many of my collections are related to tags collection, thus results many "M2M hidden collections". If I don't want them I can just create a dropdown with multiple items.

**Comment by JoshTheDerf** [comment]
This would be helpful for a different use-case: Flows. Flow trigger modals can't select relational data apart from using a collection item dropdown. If you need multiple collection item dropdowns in your flow trigger right now, you have to use a repeater + collection item dropdown interface.

**Reply by Dominic-Marcelino** [reply]
Facing the same - did you found a good solution aside from wrapping an Collection-Item-Dropdown inside a repeater? 🤔

**Comment by danmaze** [comment]
Yeah, repeater + collection item dropdown is probably the only option if you prefer not creating a separate relational collection. Note that you have to select "JSON" type when creating the field to see the collection item dropdown option.

cc @loongmxbt

**Comment by SergeyRe** [comment]
Take a look to my extension [directus-extension-keep-related-collection-ids-in-csv](https://www.npmjs.com/package/directus-extension-keep-related-collection-ids-in-csv)
Its way to keep  relation in form csv array (work both for M2M and M2O types of relation)
The main advantage of CSV field is to use it for conditions ( not possible for standard M2M)

**Comment by JamesW1** [comment]
Thanks for the suggestion! We've moved feature requests over to our public roadmap at https://roadmap.directus.io/roadmap - please submit this there instead. Thanks!
